x-pack/plugins/infra/server/services/log_views/log_views_client.ts

```diff
@@ -53,7 +53,9 @@ export class LogViewsClient implements ILogViewsClient {
     private readonly config: LogViewsStaticConfig
   ) {}
 
-  public async getLogView(logViewId: string): Promise<LogView> {
+  public async getLogView(logViewReference: LogViewReference): Promise<LogView> {
```

Comment:
**question:** there is any reason behind this signature change? Same doubt about the PUT endpoint, it adds the need to differentiate between the id property name but I don't see the benefit here, could you please explain it?

Reply:
Mmm I see, I'm just wondering if the benefit of extracting the common interface compensates for the overhead of adding this conditional logic to get the logViewReference id. 
Also, the interfaces for the client and server LogViewClient were small enough to well represent in isolation the client they have, it is probably not necessary to extract them into a common one if they have different signatures, I'd rather prefer to keep explicitly defined interfaces instead of updating the service implementation to extract a type